src/components/ConnectToDB/snippets.ts

```diff
@@ -123,27 +123,26 @@ public class YDBConnect {
 }
 
 export function getNodeJSSnippetCode({database, endpoint}: SnippetParams) {
-    return `const {Driver, getCredentialsFromEnv, getLogger} = require('ydb-sdk');
+    return `// Requires Node.js 20.19 or higher (also works with Deno/Bun)
+// Documentation: https://ydb.js.org
+// Source code: https://github.com/ydb-platform/ydb-js-sdk
+// Examples: https://github.com/ydb-platform/ydb-js-examples
+import {Driver} from '@ydbjs/core';
+import {query} from '@ydbjs/query';
+import type {ResultSet} from '@ydbjs/query';
 
-const logger = getLogger({level: 'debug'});
-const endpoint = '${endpoint ?? '<endpoint>'}';
-const database = '${database ?? '/<database>'}';
-const authService = getCredentialsFromEnv();
-const driver = new Driver({endpoint, database, authService});
+const connectionString = 'grpc://${endpoint ?? '<endpoint>'}${database ?? '/<database>'}';
```

Comment:
Please, ensure `connectionString` is properly formed.

Endpoint is a string from database `ControlPlane` without search params. Example: `grpc://klg1-0561.host.prod.ydb.yandex.net:2139`

So `connectionString` in your snippet after all transformations will be `'grpc://grpc://klg1-0561.host.prod.ydb.yandex.net:2139/Root/db1'` (assuming `/Root/db1` is our database)